opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py

```diff
@@ -26,12 +27,26 @@
 
 
 def auto_instrument():
-    for entry_point in iter_entry_points("opentelemetry_instrumentor"):
+    package_to_exclude = Configuration().get("DISABLED_INSTRUMENTATIONS", None)
+    if package_to_exclude:
+        package_to_exclude = package_to_exclude.split(",")
+        packages_to_instrument = [
+            entry_point
+            for entry_point in iter_entry_points("opentelemetry_instrumentor")
+            if entry_point.name not in package_to_exclude
+        ]
+    else:
+        packages_to_instrument = (
+            entry_point
+            for entry_point in iter_entry_points("opentelemetry_instrumentor")
+        )
```

Comment:
Could packages_to_exclude just be a list in order to get rid of this if/else case.